native/cocos/renderer/pipeline/custom/FrameGraphDispatcher.cpp

```diff
@@ -122,22 +122,70 @@ ResourceGraph::vertex_descriptor FrameGraphDispatcher::realResourceID(const ccst
     return resourceAccessGraph.resourceIndex.at(name);
 }
 
+[[nodiscard]] ccstd::pmr::string concatResName(
```

Comment:
I am not sure if `[[nodiscard]]` works on iOS11.

Reply:
`[[nodiscard]]` has been used for more than one year. I think it works.